uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkPattern.php

```diff
@@ -45,6 +45,10 @@ public static function create($params) {
       $params['weight'] = self::getMaxWeight() + 1;
     }
 
+    if(!empty($params['id']) && isset($params['is_active']) && !$params['is_active']) {
```

Comment:
That will not work here. The is_active param might not be present all the time and I want to call checkIfPatternCanBeDisabled only if it is. If I use `!empty($params['is_active'])` will return true even it is not present.